packages/component/src/SendBox/SendButton.js

```diff
@@ -10,7 +10,6 @@ const connectSendButton = (...selectors) =>
   connectToWebChat(
     ({ disabled, language, sendBoxValue, setSendBox, submitSendBox }) => ({
       click: () => {
```

Comment:
Since we are only making one function call in the `click` method, we can probably just remove it and pass `submitSendBox` to the button's `onClick` handler. Here are the changes I would recommend. 

```diff
const connectSendButton = (...selectors) =>
  connectToWebChat(
-    ({ disabled, language, sendBoxValue, setSendBox, submitSendBox }) => ({
+    ({ disabled, language, submitSendBox }) => ({
-     click: () => {
-        setSendBox(sendBoxValue.trim());
-        submitSendBox();
-     },
      disabled,
      language,
+     submitSendBox 
    }),
    ...selectors
  );

const SendButton = ({ click, disabled, language }) => (
-  <IconButton alt={localize('Send', language)} disabled={disabled} onClick={click}>
+  <IconButton alt={localize('Send', language)} disabled={disabled} onClick={submitSendBox}>
    <SendIcon />
  </IconButton>
);

SendButton.defaultProps = {
  disabled: false
};

SendButton.propTypes = {
-  click: PropTypes.func.isRequired,
  disabled: PropTypes.bool,
-  language: PropTypes.string.isRequired
+  language: PropTypes.string.isRequired,
+  submitSendBox: PropTypes.func.isRequired
};

```

Reply:
Change committed! Good eye!